web-components/graphrag-widget/src/styles.css

```
@@ -0,0 +1,539 @@
+  font-weight: 500;
+}
+.fdb-powered a:hover { text-decoration: underline; }
+
+/* ── Message Bubbles ────────────────────────────────── */
+.fdb-bubble {
+  max-width: 85%;
+  padding: 10px 14px;
+  font-size: 13px;
+  line-height: 1.55;
+  animation: fdb-msg-in 300ms cubic-bezier(0.34, 1.56, 0.64, 1);
+  word-wrap: break-word;
```

Stylelint (17.7.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)